assets/src/design-system/components/tooltip/index.js

```diff
@@ -102,8 +121,8 @@ function Tooltip({
       getBoundingBoxCenter(anchorElBoundingBox) -
       getBoundingBoxCenter(tooltipElBoundingBox);
 
-    setArrowDelta(delta);
-  }, []);
+    setArrowDelta((isMirrored ? -1 : 1) * delta);
+  }, [isMirrored]);
```

Comment:
don't absolutely love the `isMirrored` prop, but I think it's a necessary evil so that the tail placement can be calculated correctly.

Reply:
alright, looked into it and got a fix for you to not need it. so...

this is the delta, which should be directionally agnostic, just a difference between x coords. turns out it is, but the rtl plugin is multiplying all css `translateX` values by `-1`, so when we're multiplying the `delta` here by `-1` we are just reversing the reversing done by the rtl plugin. 

Another approach that would work that  wouldn't need the prop is just disabling any behavior from the rtl plugin around the translate since we're already calculating a rtl agnostic delta. I tested it out and it works, but only if you put the `/*! @noflip */` right above the translateX, so you should be able to update this function to carry it and shouldn't need the `isMirrored` prop in here any more:
`design-system/components/tooltip/tail.js: 32`
```js
const getTailPosition = ({ placement, translateX }) => {
  switch (placement) {
    case PLACEMENT.TOP:
    case PLACEMENT.TOP_START:
    case PLACEMENT.TOP_END:
      return css`
        bottom: -${TAIL_HEIGHT - 1}px;
        /*! @noflip */
        transform: translateX(${translateX}px) rotate(180deg);
      `;
    case PLACEMENT.BOTTOM:
    case PLACEMENT.BOTTOM_START:
    case PLACEMENT.BOTTOM_END:
      return css`
        top: -${TAIL_HEIGHT - 1}px;
        /*! @noflip */
        transform: translateX(${translateX}px);
      `;
    case PLACEMENT.LEFT:
    case PLACEMENT.LEFT_START:
    case PLACEMENT.LEFT_END:
      return css`
        right: -${TAIL_WIDTH / 2 + TAIL_HEIGHT / 2 - 1}px;
        transform: rotate(90deg);
      `;
    case PLACEMENT.RIGHT:
    case PLACEMENT.RIGHT_START:
    case PLACEMENT.RIGHT_END:
      return css`
        left: -${TAIL_WIDTH / 2 + TAIL_HEIGHT / 2 - 1}px;
        transform: rotate(-90deg);
      `;
    default:
      return ``;
  }
};
```

Reply:
@littlemilkstudio  yeah!!!  If you have time, could you just push this change up on this branch since @samwhale is out this week?

Reply:
super genius idea max, nice find!